pkg/service/schema.go

```diff
@@ -497,3 +537,18 @@ func (p *PlanSchemas) addCommonSchema(sp *ServiceProperties) {
 		}
 	}
 }
+
+// falsePropertySchema is used internally to deal with the fact that in JSON
+// schema, additionalProperties can be a schema or it can be the bool value
+// false.
+type falsePropertySchema struct{}
+
+// MarshalJSON provides functionality to marshal an ObjectPropertySchema to JSON
+func (falsePropertySchema) MarshalJSON() ([]byte, error) {
+	return []byte("false"), nil
+}
+
+// No-op
+func (falsePropertySchema) validate(string, interface{}) error {
+	return nil
+}
```

Comment:
See [this comment](https://github.com/Azure/open-service-broker-azure/pull/445/files#r189351592)